src/routing_table/mod.rs

```diff
@@ -1129,7 +1129,8 @@ mod tests {
         assert_eq!(our_name, table.our_name);
 
         // Get some names
-        let close_name: u32 = *unwrap!(table.our_section.iter().nth(4));
+        let close_name: u32 =
+            *unwrap!(table.our_section.iter().filter(|name| **name != our_name).nth(4));
```

Comment:
This is inside test code. The reason the filter is necessary is because `table.our_section()` used to exclude `our_name` — somehow this didn't get fixed when that method was changed (my fault I guess). The "4th" bit is obviously hacky, but I feel fairly certain it won't fail to find an element here. So you okay with `unwrap` now?